### ISSUE TITLE
Add specific mention of accessibility

### ISSUE THREAD

**Issue by monicachilders** [usnistgov/ElectionModeling#4]
On the 'set up voting stations, materials, and signage' step, we say: "the space may need to be modified to become conducive to voting."

Suggest the following: "the space may need modifications to facilitate voting or meet accessibility requirements."

*Generated from Process Portal* 
Reference Diagram: [Try to open polling location](https://pages.nist.gov/ElectionModeling/diagrams.html#Process___18_5_1_11940316_1500486634097_993849_30344)

**Comment by JDziurlaj** [comment]
I would argue that "facilitate voting" includes any accessibility requirements. I don't want there to be an implication that accessibility is a nice to have in addition to facilitating voting, but an integral part.

**Comment by monicachilders** [comment]
I think that's absolutely true if the reader is familiar with elections - this change would just make it clearer to a layperson who doesn't have that subject matter expertise. Accessibility requirements are a legal obligation rather than an optional modification, so I think it's beneficial to note that requirement explicitly vs. assuming people will know that "facilitate voting" includes it. But, as you've noted, this is a "nice-to-have" rather than a "must-have."

**Comment by JDziurlaj** [comment]
How about the following change: "the space may need modifications to facilitate voting for all voters."
